doc/languages-frameworks/rust.section.md

```diff
@@ -64,6 +65,14 @@ When the `Cargo.lock`, provided by upstream, is not in sync with the
 added in `cargoPatches` will also be prepended to the patches in `patches` at
 build-time.
 
+The `useRealVendorConfig` parameter tells cargo-vendor to include a Cargo
+configuration file in the fetched dependencies. This will fix problems with
+projects, where Crates are downloaded from non-crates.io sources. Please note,
+that currently this parameter defaults to `false` only due to compatibility
+reasons, as setting this to `true` requires a change in `cargoSha256`.
+Eventually this distinction will be deprecated, so please always set
+`useRealVendorConfig` to `true` and make sure to recompute the `cargoSha256`.
```

Comment:
For this to become the default I would like to see our own script generating `.cargo/config` to avoid breaking checksums.
That's why I would not set it by default.